fe2-android/app/src/test/unit/java/com/github/dedis/popstellar/model/network/method/message/data/consensus/ConsensusAcceptTest.java

```diff
@@ -28,11 +26,6 @@ public class ConsensusAcceptTest {
   private static final ConsensusAccept accept =
       new ConsensusAccept(instanceId, messageId, timeInSeconds, acceptedTry, acceptedValue);
 
-  @Before
-  public void setup() {
-    JsonTestUtils.loadGSON(ApplicationProvider.getApplicationContext());
-  }
-
```

Comment:
I'm curious, why isn't it needed anymore ?

Reply:
In the first place I put the setup method as the loadGson was using the application context, then I realised that I didn't need the context there, so it was better to have a static centralised initialisation over than multiple setups. This was also generating multiple instances of databases so not being optimal